include/openssl/ec_key.h

```diff
@@ -312,6 +312,16 @@ OPENSSL_EXPORT EC_KEY *d2i_ECParameters(EC_KEY **out_key, const uint8_t **inp,
 // are supported.
 OPENSSL_EXPORT int i2d_ECParameters(const EC_KEY *key, uint8_t **outp);
 
+// d2i_ECPKParameters_bio deserializes the |ECPKParameters| specified in RFC
+// 3279 to an |EC_GROUP| from |bio|. Only deserialization of namedCurves or
+// explicitly-encoded versions of namedCurves are supported.
+OPENSSL_EXPORT EC_GROUP *d2i_ECPKParameters_bio(BIO *bio, EC_GROUP **out_group);
```

Comment:
The caller should initialize (`out_group = NULL`), otherwise `d2i_ECPKParameters` assumes [`*out_group` needs to be freed](https://github.com/aws/aws-lc/blob/ce7b0e7db247f3a27e84fa5d6e4014a07d21b540/crypto/ec_extra/ec_asn1.c#L512)